libraries/chain/controller.cpp

```diff
@@ -581,6 +582,21 @@ rpc::chain::read_contract_response controller_impl::read_contract( const rpc::ch
    };
 }
 
+rpc::chain::get_account_nonce_response controller_impl::get_account_nonce( const rpc::chain::get_account_nonce_request& request )
+{
+   apply_context ctx;
+
+   ctx.push_frame( koinos::chain::stack_frame {
+      .call = crypto::hash( CRYPTO_RIPEMD160_ID, "get_fork_data"s ).digest,
```

Comment:
Is `get_fork_data` a copy paste error here?

Reply:
yep. It doesn't actually matter what we put here, but I'll make a change.